vaadin-combo-box-flow-parent/vaadin-combo-box-flow-integration-tests/src/test/java/com/vaadin/flow/component/combobox/test/dataview/AbstractItemCountComboBoxIT.java

```diff
@@ -32,7 +32,7 @@ public abstract class AbstractItemCountComboBoxIT extends AbstractComboBoxIT {
 
     // changing the dimension might get combo box change what it fetches and
     // how many items it shows, so changing this is a bad idea ...
-    private static final Dimension TARGET_SIZE = new Dimension(1000, 900);
+    private static final Dimension TARGET_SIZE = new Dimension(1000, 1100);
```

Comment:
This needs to be larger now, probably because the browser frame takes up some height.